job_templates/ansible_roles_-_ansible_default.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+name: Ansible Roles - Ansible Default
+job_category: Ansible Playbook
+description_format: Run Ansible roles
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - debug: msg="Parameters provided: {{ foreman_params }}"
+  roles:
+    <% if @host.all_ansible_roles.present? -%>
+    - <%= @host.all_ansible_roles.map { |role| role.name.strip }.join("\n    - ") %>
```

Comment:
since you're already doing map, how about `map { |role| "    - #{role.name.strip}" }.join("\n")` to avoid prefix duplication? then you could also write it without hardcoding whitespaces into erb, see https://github.com/theforeman/community-templates/blob/develop/provisioning_templates/snippet/kickstart_networking_setup.erb for example of what I mean